examples/plot_neural_coding_reward_example.py

```diff
@@ -0,0 +1,307 @@
+"""
+============================================
+Neural Coding Reward Example
+============================================
+
+Reproduce a figure from a published article with a public dataset
+
+"""
+
+########################################################
+
+import matplotlib.pyplot as plt
+plt.style.use('seaborn-ticks')
+plt.switch_backend('Agg')
```

Comment:
this shouldn't be there in an example. If you don't want to build all examples locally you should use `make html-noplot`. For tests, you can use `Qt4Agg` backend